### DISCUSSION TITLE
[implemented] List more tokens by default

### DISCUSSION THREAD

**Discussion by wjmelements** [airswap/airswap-web#146] [Ideas]
The default list should probably be much larger. Can populate with the intersection of a (Uniswap-style) tokenlist and the Registry.

**Comment by gpxl-dev** [comment]
This should be fixed now  - you'll need to [clear your localStorage](https://developer.chrome.com/docs/devtools/storage/localstorage/#deleteall) to get the new behaviour though.

There are a couple of tokens with no symbol which render an ugly placeholder right now - we'll sort this!
